app/services/nomis/offender.rb

```diff
@@ -27,9 +27,15 @@ def imprisonment_status
 private
 
   def details
-    @details ||= Nomis::Api.instance.
-                   lookup_offender_details(noms_id: noms_id)
-  rescue Nomis::APIError
+    @details ||= Nomis::Api.instance.lookup_offender_details(noms_id: noms_id)
+  rescue Nomis::DisabledError
```

Comment:
This is an interesting one ...
The 'process_a_request_nomis_disabled_spec.rb' switches off the nomis api by returning false when calling 'enabled?'. The initialiser will throw a Nomis::DisabledError if the api is not enabled. Therefore we need to resuce from this exception in order to not break the test :-( 
Would be worth revisiting this logic when Yann returns